ui/src/Groups/Messages/Messages.scss

```diff
@@ -0,0 +1,21 @@
+.Messages {
+  display: flex;
+
+  &--consumer {
+    flex-direction: column;
+  }
+
+  .Message {
```

Comment:
Message css in Messages? Should this not be in Message.scss (plus, this isnt BEM)

Reply:
I have moved some of it over to Message.scss. The last part is for the case where we have multiple `Message` elements in a list and we need to ensure they all line up. I have BEM-ified that part, hopefully it's correct now!